frontend/src/app/sessions/license-indicator/license-indicator.component.html

```diff
@@ -0,0 +1,38 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+@if ((licenseUsageWrapperService.licenseServerUsage$ | async) !== undefined) {
+  @for (
+    licenseServer of licenseUsageWrapperService.licenseServerUsage$ | async;
+    track licenseServer.id
+  ) {
+    @if (licenseServer.usage.free === 0) {
+      <div
+        class="my-1 flex items-center gap-1.5 rounded border bg-red-500 p-1 text-sm text-white shadow"
+      >
+        <mat-icon class="shrink-0">error</mat-icon>
+        <span
+          >All {{ licenseServer.usage.total }} TeamForCapella licenses are
+          currently in use. You will not be able to start a new session until a
+          license is available. Please make sure to terminate your sessions
+          after use.
```

Comment:
```suggestion
          >All TeamForCapella licenses are
          currently in use. You will still be able to start a new session, but you may not be able to connect to TeamForCapella, the connection might be refused with "Invalid license". Please make sure to terminate your sessions
          after use.
```

Users should still be able to request sessions on their own risk (there are also users who don't use TeamForCapella).